app/src/main/java/org/wikipedia/Constants.java

```diff
@@ -58,7 +59,10 @@
     public enum InvokeSource {
         PAGE_ACTIVITY,
         EDIT_FEED_TITLE_DESC,
-        EDIT_FEED_TRANSLATE_TITLE_DESC
+        EDIT_FEED_TRANSLATE_TITLE_DESC,
+        FEED_CARD_SUGGESTED_EDITS_ADD_DESC,
+        FEED_CARD_SUGGESTED_EDITS_TRANSLATE_DESC,
```

Comment:
Does there have a strong reason to create two pretty much the same InvokeSource? Otherwise, I feel that we can use the existing ones.

Reply:
It is used in DescriptionEditActivito to determine if it is a translations. I could have just sent the 'isTranslation' boolean that i use in the cardView... but I feel this is more readable..